en/metadata.yaml

```diff
@@ -36,6 +36,13 @@ navigation:
       - title: Introduction
         location: stacks/index.md
 
+  - title: Manage a device
```

Comment:
![screenshot from 2017-08-16 11-11-36](https://user-images.githubusercontent.com/18480003/29356174-b365f8a0-8273-11e7-83da-afca937a5c72.png)

I understand the rationale, but I think it makes the navigation rather odd since every item under "Manage a device", after this one, is a child of "Build a stack".